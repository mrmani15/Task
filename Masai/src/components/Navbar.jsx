import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class Navbar extends Component {
	constructor(props) {
		super(props);

		this.state = {
			searchName:''
		};
	}

	handleChange = e => {
		this.setState({
			searchName: e.target.value
		})
	}

	handleClick = e => {
		e.preventDefault();
		console.log(this.state.searchName)
		this.setState({
			searchName: ''
		})
	}

	render() {
		return (
			<div
				style={{
					borderBottom: '1px grey solid',
					width: '100%',
					height: '60px',
				}}
			>
				<div className="container pt-1">
					<div className="row d-flex align-items-center">
						<div className="col-8">
							<div className="row d-flex align-items-center">
								<div className="col-1">
									<img
										src="/logo.png"
										alt="logo"
										style={{ width: '50px' }}
									/>
								</div>
								<div className="input-group rounded-pill col-9">
									<input
										type="text"
										className="form-control rounded-pill"
										placeholder="Search Twitter"
										name='searchName'
										onChange={this.handleChange}
										value={this.state.searchName}
									/>
								</div>
								<div>
									<button className='btn btn-primary rounded-pill' onClick={this.handleClick}>
											Search
									</button>
								</div>
							</div>
						</div>
						<div className="col-4">
							<div className="row d-flex align-items-center">
								<div className="col">
									<Link to="/sign-up">
										<button className="btn btn-primary border rounded-pill form-control">
											Sign Up
										</button>
									</Link>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		);
	}
}

export default Navbar;
