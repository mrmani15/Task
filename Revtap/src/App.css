.App {
	text-align: center;
}
li {
	font-size: 1.4rem;
}
