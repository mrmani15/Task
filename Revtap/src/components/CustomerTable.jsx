import React from 'react';

function CustomerTable({ currentPosts }) {
	if (currentPosts) {
		return (
			<div>
				<table className="table table-striped table-secondary">
					<thead>
						<tr>
							<th scope="col">ID</th>
							<th scope="col">First Name</th>
							<th scope="col">Last Name</th>
							<th scope="col">Email</th>
							<th scope="col">Created</th>
							<th scope="col">Orders</th>
						</tr>
					</thead>
					<tbody>
						{currentPosts.map(ele => {
							return (
								<tr key={ele.id}>
									<td>{ele.id}</td>
									<td>{ele.firstName}</td>
									<td>{ele.lastName}</td>
									<td>{ele.email}</td>
									<td>{ele.created}</td>
									<td>{ele.orders}</td>
								</tr>
							);
						})}
					</tbody>
				</table>
			</div>
		);
	} else {
		return <h1>Loading</h1>;
	}
}

export default CustomerTable;
