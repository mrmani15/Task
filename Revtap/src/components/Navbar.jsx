import React from 'react';
import { Link } from 'react-router-dom';

function Navbar() {
	return (
		<>
			<ul className="nav justify-content-center container bg-light mt-2">
				<li className="nav-item m-auto">
					<Link to="/" className="nav-link">
						CUSTOMER TABLE
					</Link>
				</li>
				<li className="nav-item m-auto">
					<Link to="/price-graph" className="nav-link">
						PRICE GRAPH
					</Link>
				</li>
				<li className="nav-item m-auto">
					<Link to="/order-graph" className="nav-link">
						ORDER GRAPH
					</Link>
				</li>
			</ul>
		</>
	);
}

export default Navbar;
