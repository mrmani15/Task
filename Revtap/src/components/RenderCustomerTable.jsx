import React, { useState } from 'react';
import data from '../util/data.json';
import Pagination from './Pagination';
import CustomerTable from './CustomerTable';

function RenderCustomerTable() {
	const [Data] = useState([data.customers]);
	const [currentPage, setCurrentPage] = useState(1);
	const [postsPerPage] = useState(10);

	const indexOfLastPost = currentPage * postsPerPage;
	const indexOfFirstPost = indexOfLastPost - postsPerPage;
	const currentPosts = Data[0].slice(indexOfFirstPost, indexOfLastPost);

	const paginate = pageNumber => setCurrentPage(pageNumber);

	return (
		<div className="container mt-3">
			<CustomerTable currentPosts={currentPosts} />
			<Pagination
				postsPerPage={postsPerPage}
				totalPosts={Data[0].length}
				paginate={paginate}
			/>
		</div>
	);
}

export default RenderCustomerTable;
