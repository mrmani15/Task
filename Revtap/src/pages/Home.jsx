import React from 'react';
import RenderCustomerTable from '../components/RenderCustomerTable';

function Home() {
	return (
		<div>
			<RenderCustomerTable />
		</div>
	);
}

export default Home;
