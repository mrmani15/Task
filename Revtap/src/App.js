import React from 'react';
import './App.css';
import { BrowserRouter } from 'react-router-dom'
import Home from './pages/Home';
import PriceCount from './components/PriceCount';
import OrderCount from './components/OrderCount';
import Navbar from './components/Navbar';


function App() {
  return (
    <BrowserRouter>
      <div className="App">
        <Navbar />
        <Home />
        <PriceCount />
        <OrderCount />
      </div>
    </BrowserRouter>
  );
}

export default App;
