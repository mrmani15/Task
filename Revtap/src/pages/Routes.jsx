import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import Home from './Home';
import PriceCount from '../components/PriceCount';
import OrderCount from '../components/OrderCount';
import Navbar from '../components/Navbar';

function Routes() {
	return (
		<BrowserRouter>
			<Navbar />
			<Switch>
				<Route path="/" exact component={Home} />
				<Route path="/price-graph" component={PriceCount} />
				<Route path="/order-graph" component={OrderCount} />
			</Switch>
		</BrowserRouter>
	);
}

export default Routes;
