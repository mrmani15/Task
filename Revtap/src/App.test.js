import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';
import Home from './pages/Home';
import Navbar from './components/Navbar';
import Pagination from './components/Pagination';
import CustomerTable from './components/CustomerTable';
import Routes from './pages/Routes';
import RenderCustomerTable from './components/RenderCustomerTable';


it('snapshot for App component', () => {
  const tree = renderer.create(<App />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('snapshot for Home component', () => {
  const tree = renderer.create(<Home />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('snapshot for navbar component', () => {
  const tree = renderer.create(<Navbar />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('snapshot for pagination component', () => {
  const tree = renderer.create(<Pagination />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('snapshot for customerTable component', () => {
  const tree = renderer.create(<CustomerTable />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('snapshot for Routes component', () => {
  const tree = renderer.create(<Routes />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('snapshot for Table component', () => {
  const tree = renderer.create(<RenderCustomerTable />).toJSON();

  expect(tree).toMatchSnapshot();
});