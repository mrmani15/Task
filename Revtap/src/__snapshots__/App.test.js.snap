// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for App component 1`] = `
<div
  className="App"
>
  <ul
    className="nav justify-content-center container bg-light mt-2"
  >
    <li
      className="nav-item m-auto"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
        CUSTOMER TABLE
      </a>
    </li>
    <li
      className="nav-item m-auto"
    >
      <a
        className="nav-link"
        href="/price-graph"
        onClick={[Function]}
      >
        PRICE GRAPH
      </a>
    </li>
    <li
      className="nav-item m-auto"
    >
      <a
        className="nav-link"
        href="/order-graph"
        onClick={[Function]}
      >
        ORDER GRAPH
      </a>
    </li>
  </ul>
  <div>
    <div
      className="container mt-3"
    >
      <div>
        <table
          className="table table-striped table-secondary"
        >
          <thead>
            <tr>
              <th
                scope="col"
              >
                ID
              </th>
              <th
                scope="col"
              >
                First Name
              </th>
              <th
                scope="col"
              >
                Last Name
              </th>
              <th
                scope="col"
              >
                Email
              </th>
              <th
                scope="col"
              >
                Created
              </th>
              <th
                scope="col"
              >
                Orders
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                0b62a22a-51b5-4594-921a-e25a12ece0b8
              </td>
              <td>
                Dion
              </td>
              <td>
                Donnelly
              </td>
              <td>
                Carmella_Willms@gmail.com
              </td>
              <td>
                2020-04-23T21:35:28.635Z
              </td>
              <td>
                2
              </td>
            </tr>
            <tr>
              <td>
                0fcbff30-5e8d-4ffb-949e-c7f1301c71e1
              </td>
              <td>
                Diego
              </td>
              <td>
                Hahn
              </td>
              <td>
                Fatima_Larson@yahoo.com
              </td>
              <td>
                2020-04-02T04:50:54.591Z
              </td>
              <td>
                0
              </td>
            </tr>
            <tr>
              <td>
                621777e6-e862-474b-a2db-5812d848cedc
              </td>
              <td>
                Reinhold
              </td>
              <td>
                Ondricka
              </td>
              <td>
                Gus17@yahoo.com
              </td>
              <td>
                2020-04-04T03:29:24.579Z
              </td>
              <td>
                2
              </td>
            </tr>
            <tr>
              <td>
                2b6bb67e-fb90-4eb6-af7b-e85945fe40ad
              </td>
              <td>
                Mary
              </td>
              <td>
                Bernhard
              </td>
              <td>
                Layne67@gmail.com
              </td>
              <td>
                2020-04-26T05:05:10.164Z
              </td>
              <td>
                0
              </td>
            </tr>
            <tr>
              <td>
                6c51fb63-eae9-4b72-b365-7db31612a543
              </td>
              <td>
                Napoleon
              </td>
              <td>
                Beahan
              </td>
              <td>
                Nia.Streich17@yahoo.com
              </td>
              <td>
                2020-04-09T01:39:29.883Z
              </td>
              <td>
                1
              </td>
            </tr>
            <tr>
              <td>
                72598b8a-33ab-48ce-8539-f4e2d3fd7ae5
              </td>
              <td>
                Joseph
              </td>
              <td>
                Jacobs
              </td>
              <td>
                Rozella22@yahoo.com
              </td>
              <td>
                2020-04-22T22:43:24.254Z
              </td>
              <td>
                1
              </td>
            </tr>
            <tr>
              <td>
                eea3a83f-99c7-4434-b3f5-e52679f1f682
              </td>
              <td>
                Chase
              </td>
              <td>
                Ryan
              </td>
              <td>
                Arden20@hotmail.com
              </td>
              <td>
                2020-04-08T08:44:51.927Z
              </td>
              <td>
                0
              </td>
            </tr>
            <tr>
              <td>
                07ae0e34-9dac-4c32-bf03-39f8e79f5f86
              </td>
              <td>
                Doyle
              </td>
              <td>
                Grant
              </td>
              <td>
                Ruthie_Glover30@hotmail.com
              </td>
              <td>
                2020-04-26T10:47:51.802Z
              </td>
              <td>
                1
              </td>
            </tr>
            <tr>
              <td>
                21419e19-48c6-420a-af0f-a501b5cbbef4
              </td>
              <td>
                Dolly
              </td>
              <td>
                Schmeler
              </td>
              <td>
                Lori.Roob4@gmail.com
              </td>
              <td>
                2020-04-14T04:10:23.744Z
              </td>
              <td>
                2
              </td>
            </tr>
            <tr>
              <td>
                d046c38d-4b7c-463e-b941-80ac0d43084c
              </td>
              <td>
                Maritza
              </td>
              <td>
                Kunde
              </td>
              <td>
                Demetris_Tromp@hotmail.com
              </td>
              <td>
                2020-04-20T20:27:36.295Z
              </td>
              <td>
                3
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      <nav>
        <ul
          className="pagination"
        >
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              1
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              2
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              3
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              4
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              5
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              6
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              7
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              8
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              9
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              10
            </div>
          </li>
        </ul>
      </nav>
    </div>
  </div>
</div>
`;

exports[`snapshot for Home component 1`] = `
<div>
  <div
    className="container mt-3"
  >
    <div>
      <table
        className="table table-striped table-secondary"
      >
        <thead>
          <tr>
            <th
              scope="col"
            >
              ID
            </th>
            <th
              scope="col"
            >
              First Name
            </th>
            <th
              scope="col"
            >
              Last Name
            </th>
            <th
              scope="col"
            >
              Email
            </th>
            <th
              scope="col"
            >
              Created
            </th>
            <th
              scope="col"
            >
              Orders
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              0b62a22a-51b5-4594-921a-e25a12ece0b8
            </td>
            <td>
              Dion
            </td>
            <td>
              Donnelly
            </td>
            <td>
              Carmella_Willms@gmail.com
            </td>
            <td>
              2020-04-23T21:35:28.635Z
            </td>
            <td>
              2
            </td>
          </tr>
          <tr>
            <td>
              0fcbff30-5e8d-4ffb-949e-c7f1301c71e1
            </td>
            <td>
              Diego
            </td>
            <td>
              Hahn
            </td>
            <td>
              Fatima_Larson@yahoo.com
            </td>
            <td>
              2020-04-02T04:50:54.591Z
            </td>
            <td>
              0
            </td>
          </tr>
          <tr>
            <td>
              621777e6-e862-474b-a2db-5812d848cedc
            </td>
            <td>
              Reinhold
            </td>
            <td>
              Ondricka
            </td>
            <td>
              Gus17@yahoo.com
            </td>
            <td>
              2020-04-04T03:29:24.579Z
            </td>
            <td>
              2
            </td>
          </tr>
          <tr>
            <td>
              2b6bb67e-fb90-4eb6-af7b-e85945fe40ad
            </td>
            <td>
              Mary
            </td>
            <td>
              Bernhard
            </td>
            <td>
              Layne67@gmail.com
            </td>
            <td>
              2020-04-26T05:05:10.164Z
            </td>
            <td>
              0
            </td>
          </tr>
          <tr>
            <td>
              6c51fb63-eae9-4b72-b365-7db31612a543
            </td>
            <td>
              Napoleon
            </td>
            <td>
              Beahan
            </td>
            <td>
              Nia.Streich17@yahoo.com
            </td>
            <td>
              2020-04-09T01:39:29.883Z
            </td>
            <td>
              1
            </td>
          </tr>
          <tr>
            <td>
              72598b8a-33ab-48ce-8539-f4e2d3fd7ae5
            </td>
            <td>
              Joseph
            </td>
            <td>
              Jacobs
            </td>
            <td>
              Rozella22@yahoo.com
            </td>
            <td>
              2020-04-22T22:43:24.254Z
            </td>
            <td>
              1
            </td>
          </tr>
          <tr>
            <td>
              eea3a83f-99c7-4434-b3f5-e52679f1f682
            </td>
            <td>
              Chase
            </td>
            <td>
              Ryan
            </td>
            <td>
              Arden20@hotmail.com
            </td>
            <td>
              2020-04-08T08:44:51.927Z
            </td>
            <td>
              0
            </td>
          </tr>
          <tr>
            <td>
              07ae0e34-9dac-4c32-bf03-39f8e79f5f86
            </td>
            <td>
              Doyle
            </td>
            <td>
              Grant
            </td>
            <td>
              Ruthie_Glover30@hotmail.com
            </td>
            <td>
              2020-04-26T10:47:51.802Z
            </td>
            <td>
              1
            </td>
          </tr>
          <tr>
            <td>
              21419e19-48c6-420a-af0f-a501b5cbbef4
            </td>
            <td>
              Dolly
            </td>
            <td>
              Schmeler
            </td>
            <td>
              Lori.Roob4@gmail.com
            </td>
            <td>
              2020-04-14T04:10:23.744Z
            </td>
            <td>
              2
            </td>
          </tr>
          <tr>
            <td>
              d046c38d-4b7c-463e-b941-80ac0d43084c
            </td>
            <td>
              Maritza
            </td>
            <td>
              Kunde
            </td>
            <td>
              Demetris_Tromp@hotmail.com
            </td>
            <td>
              2020-04-20T20:27:36.295Z
            </td>
            <td>
              3
            </td>
          </tr>
        </tbody>
      </table>
    </div>
    <nav>
      <ul
        className="pagination"
      >
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            1
          </div>
        </li>
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            2
          </div>
        </li>
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            3
          </div>
        </li>
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            4
          </div>
        </li>
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            5
          </div>
        </li>
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            6
          </div>
        </li>
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            7
          </div>
        </li>
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            8
          </div>
        </li>
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            9
          </div>
        </li>
        <li
          className="page-item"
        >
          <div
            className="page-link"
            onClick={[Function]}
          >
            10
          </div>
        </li>
      </ul>
    </nav>
  </div>
</div>
`;

exports[`snapshot for Routes component 1`] = `
Array [
  <ul
    className="nav justify-content-center container bg-light mt-2"
  >
    <li
      className="nav-item m-auto"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
        CUSTOMER TABLE
      </a>
    </li>
    <li
      className="nav-item m-auto"
    >
      <a
        className="nav-link"
        href="/price-graph"
        onClick={[Function]}
      >
        PRICE GRAPH
      </a>
    </li>
    <li
      className="nav-item m-auto"
    >
      <a
        className="nav-link"
        href="/order-graph"
        onClick={[Function]}
      >
        ORDER GRAPH
      </a>
    </li>
  </ul>,
  <div>
    <div
      className="container mt-3"
    >
      <div>
        <table
          className="table table-striped table-secondary"
        >
          <thead>
            <tr>
              <th
                scope="col"
              >
                ID
              </th>
              <th
                scope="col"
              >
                First Name
              </th>
              <th
                scope="col"
              >
                Last Name
              </th>
              <th
                scope="col"
              >
                Email
              </th>
              <th
                scope="col"
              >
                Created
              </th>
              <th
                scope="col"
              >
                Orders
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                0b62a22a-51b5-4594-921a-e25a12ece0b8
              </td>
              <td>
                Dion
              </td>
              <td>
                Donnelly
              </td>
              <td>
                Carmella_Willms@gmail.com
              </td>
              <td>
                2020-04-23T21:35:28.635Z
              </td>
              <td>
                2
              </td>
            </tr>
            <tr>
              <td>
                0fcbff30-5e8d-4ffb-949e-c7f1301c71e1
              </td>
              <td>
                Diego
              </td>
              <td>
                Hahn
              </td>
              <td>
                Fatima_Larson@yahoo.com
              </td>
              <td>
                2020-04-02T04:50:54.591Z
              </td>
              <td>
                0
              </td>
            </tr>
            <tr>
              <td>
                621777e6-e862-474b-a2db-5812d848cedc
              </td>
              <td>
                Reinhold
              </td>
              <td>
                Ondricka
              </td>
              <td>
                Gus17@yahoo.com
              </td>
              <td>
                2020-04-04T03:29:24.579Z
              </td>
              <td>
                2
              </td>
            </tr>
            <tr>
              <td>
                2b6bb67e-fb90-4eb6-af7b-e85945fe40ad
              </td>
              <td>
                Mary
              </td>
              <td>
                Bernhard
              </td>
              <td>
                Layne67@gmail.com
              </td>
              <td>
                2020-04-26T05:05:10.164Z
              </td>
              <td>
                0
              </td>
            </tr>
            <tr>
              <td>
                6c51fb63-eae9-4b72-b365-7db31612a543
              </td>
              <td>
                Napoleon
              </td>
              <td>
                Beahan
              </td>
              <td>
                Nia.Streich17@yahoo.com
              </td>
              <td>
                2020-04-09T01:39:29.883Z
              </td>
              <td>
                1
              </td>
            </tr>
            <tr>
              <td>
                72598b8a-33ab-48ce-8539-f4e2d3fd7ae5
              </td>
              <td>
                Joseph
              </td>
              <td>
                Jacobs
              </td>
              <td>
                Rozella22@yahoo.com
              </td>
              <td>
                2020-04-22T22:43:24.254Z
              </td>
              <td>
                1
              </td>
            </tr>
            <tr>
              <td>
                eea3a83f-99c7-4434-b3f5-e52679f1f682
              </td>
              <td>
                Chase
              </td>
              <td>
                Ryan
              </td>
              <td>
                Arden20@hotmail.com
              </td>
              <td>
                2020-04-08T08:44:51.927Z
              </td>
              <td>
                0
              </td>
            </tr>
            <tr>
              <td>
                07ae0e34-9dac-4c32-bf03-39f8e79f5f86
              </td>
              <td>
                Doyle
              </td>
              <td>
                Grant
              </td>
              <td>
                Ruthie_Glover30@hotmail.com
              </td>
              <td>
                2020-04-26T10:47:51.802Z
              </td>
              <td>
                1
              </td>
            </tr>
            <tr>
              <td>
                21419e19-48c6-420a-af0f-a501b5cbbef4
              </td>
              <td>
                Dolly
              </td>
              <td>
                Schmeler
              </td>
              <td>
                Lori.Roob4@gmail.com
              </td>
              <td>
                2020-04-14T04:10:23.744Z
              </td>
              <td>
                2
              </td>
            </tr>
            <tr>
              <td>
                d046c38d-4b7c-463e-b941-80ac0d43084c
              </td>
              <td>
                Maritza
              </td>
              <td>
                Kunde
              </td>
              <td>
                Demetris_Tromp@hotmail.com
              </td>
              <td>
                2020-04-20T20:27:36.295Z
              </td>
              <td>
                3
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      <nav>
        <ul
          className="pagination"
        >
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              1
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              2
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              3
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              4
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              5
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              6
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              7
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              8
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              9
            </div>
          </li>
          <li
            className="page-item"
          >
            <div
              className="page-link"
              onClick={[Function]}
            >
              10
            </div>
          </li>
        </ul>
      </nav>
    </div>
  </div>,
]
`;

exports[`snapshot for Table component 1`] = `
<div
  className="container mt-3"
>
  <div>
    <table
      className="table table-striped table-secondary"
    >
      <thead>
        <tr>
          <th
            scope="col"
          >
            ID
          </th>
          <th
            scope="col"
          >
            First Name
          </th>
          <th
            scope="col"
          >
            Last Name
          </th>
          <th
            scope="col"
          >
            Email
          </th>
          <th
            scope="col"
          >
            Created
          </th>
          <th
            scope="col"
          >
            Orders
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            0b62a22a-51b5-4594-921a-e25a12ece0b8
          </td>
          <td>
            Dion
          </td>
          <td>
            Donnelly
          </td>
          <td>
            Carmella_Willms@gmail.com
          </td>
          <td>
            2020-04-23T21:35:28.635Z
          </td>
          <td>
            2
          </td>
        </tr>
        <tr>
          <td>
            0fcbff30-5e8d-4ffb-949e-c7f1301c71e1
          </td>
          <td>
            Diego
          </td>
          <td>
            Hahn
          </td>
          <td>
            Fatima_Larson@yahoo.com
          </td>
          <td>
            2020-04-02T04:50:54.591Z
          </td>
          <td>
            0
          </td>
        </tr>
        <tr>
          <td>
            621777e6-e862-474b-a2db-5812d848cedc
          </td>
          <td>
            Reinhold
          </td>
          <td>
            Ondricka
          </td>
          <td>
            Gus17@yahoo.com
          </td>
          <td>
            2020-04-04T03:29:24.579Z
          </td>
          <td>
            2
          </td>
        </tr>
        <tr>
          <td>
            2b6bb67e-fb90-4eb6-af7b-e85945fe40ad
          </td>
          <td>
            Mary
          </td>
          <td>
            Bernhard
          </td>
          <td>
            Layne67@gmail.com
          </td>
          <td>
            2020-04-26T05:05:10.164Z
          </td>
          <td>
            0
          </td>
        </tr>
        <tr>
          <td>
            6c51fb63-eae9-4b72-b365-7db31612a543
          </td>
          <td>
            Napoleon
          </td>
          <td>
            Beahan
          </td>
          <td>
            Nia.Streich17@yahoo.com
          </td>
          <td>
            2020-04-09T01:39:29.883Z
          </td>
          <td>
            1
          </td>
        </tr>
        <tr>
          <td>
            72598b8a-33ab-48ce-8539-f4e2d3fd7ae5
          </td>
          <td>
            Joseph
          </td>
          <td>
            Jacobs
          </td>
          <td>
            Rozella22@yahoo.com
          </td>
          <td>
            2020-04-22T22:43:24.254Z
          </td>
          <td>
            1
          </td>
        </tr>
        <tr>
          <td>
            eea3a83f-99c7-4434-b3f5-e52679f1f682
          </td>
          <td>
            Chase
          </td>
          <td>
            Ryan
          </td>
          <td>
            Arden20@hotmail.com
          </td>
          <td>
            2020-04-08T08:44:51.927Z
          </td>
          <td>
            0
          </td>
        </tr>
        <tr>
          <td>
            07ae0e34-9dac-4c32-bf03-39f8e79f5f86
          </td>
          <td>
            Doyle
          </td>
          <td>
            Grant
          </td>
          <td>
            Ruthie_Glover30@hotmail.com
          </td>
          <td>
            2020-04-26T10:47:51.802Z
          </td>
          <td>
            1
          </td>
        </tr>
        <tr>
          <td>
            21419e19-48c6-420a-af0f-a501b5cbbef4
          </td>
          <td>
            Dolly
          </td>
          <td>
            Schmeler
          </td>
          <td>
            Lori.Roob4@gmail.com
          </td>
          <td>
            2020-04-14T04:10:23.744Z
          </td>
          <td>
            2
          </td>
        </tr>
        <tr>
          <td>
            d046c38d-4b7c-463e-b941-80ac0d43084c
          </td>
          <td>
            Maritza
          </td>
          <td>
            Kunde
          </td>
          <td>
            Demetris_Tromp@hotmail.com
          </td>
          <td>
            2020-04-20T20:27:36.295Z
          </td>
          <td>
            3
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <nav>
    <ul
      className="pagination"
    >
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          1
        </div>
      </li>
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          2
        </div>
      </li>
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          3
        </div>
      </li>
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          4
        </div>
      </li>
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          5
        </div>
      </li>
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          6
        </div>
      </li>
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          7
        </div>
      </li>
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          8
        </div>
      </li>
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          9
        </div>
      </li>
      <li
        className="page-item"
      >
        <div
          className="page-link"
          onClick={[Function]}
        >
          10
        </div>
      </li>
    </ul>
  </nav>
</div>
`;

exports[`snapshot for customerTable component 1`] = `
<h1>
  Loading
</h1>
`;

exports[`snapshot for navbar component 1`] = `
<ul
  className="nav justify-content-center container bg-light mt-2"
>
  <li
    className="nav-item m-auto"
  >
    <a
      className="nav-link"
      href="/"
      onClick={[Function]}
    >
      CUSTOMER TABLE
    </a>
  </li>
  <li
    className="nav-item m-auto"
  >
    <a
      className="nav-link"
      href="/price-graph"
      onClick={[Function]}
    >
      PRICE GRAPH
    </a>
  </li>
  <li
    className="nav-item m-auto"
  >
    <a
      className="nav-link"
      href="/order-graph"
      onClick={[Function]}
    >
      ORDER GRAPH
    </a>
  </li>
</ul>
`;

exports[`snapshot for pagination component 1`] = `
<nav>
  <ul
    className="pagination"
  />
</nav>
`;
